code/obj/flock/structure/relay.dm

```diff
@@ -178,6 +179,7 @@ TYPEINFO(/obj/flock_structure/relay)
 			for(var/y = -2 to 2)
 				flockdronegibs(locate(location.x + x, location.y + y, location.z))
 		explosion_new(src, location, 2000)
+		src.flock.relay_finished = TRUE
```

Comment:
what's the logic for moving this down here?

Reply:
It's to take into account that once the relay starts broadcasting it actually takes some time to go off, alternatively could just make it invincible